README.md

```diff
@@ -34,26 +34,22 @@ $ npm install dvlp
 ## Usage
 
 ```text
-$ dvlp --help
-
 Usage: dvlp [options] [path...]
 
 Start a development server, restarting and reloading connected browsers on file changes.
   Serves static files from one or more "path" directories, or a custom application
   server if "path" is a single application server file.
 
 Options:
-  -p, --port <port>           port number
-  -m, --mock <path>           path to mock files (directory, file, glob pattern)
-  -k, --hooks <path>          path to optional hooks registration file
-  --ssl <path>                enable https mode by specifying path to
-                              ".crt" and ".key" files (directory, glob pattern)
-  -r, --rollup-config <path>  path to optional Rollup.js config file
-  -s, --silent                suppress default logging
-  --no-reload                 disable reloading connected browsers on file change
-  -t, --transpiler <path>     [deprecated] path to optional transpiler file
-  -v, --version               output the version number
-  -h, --help                  display help for command
+  -p, --port <port>   port number
+  -m, --mock <path>   path to mock files (directory, file, glob pattern)
+  -k, --hooks <path>  path to optional hooks registration file
+  --ssl <path>        enable https mode by specifying path to directory containing
+                      ".crt" and ".key" files
```

Comment:
Should we add back "(directory or glob pattern)"?

Reply:
Ah, sorry. This was generated by cli `dvlp --help`. Will add to both...